grafana-plugin/src/components/PluginLink/routes.tsx

```diff
@@ -0,0 +1,113 @@
+import { AppRootProps } from '@grafana/data';
```

Comment:
I did not get why such kinda global routes config is located in `components/PluginLink` directory

Reply:
What import are you referring to? You've highlighted `@grafana/data`